### ISSUE TITLE
HMW make tag styling consistent

### ISSUE THREAD

**Issue by fannyBkane** [emfoundation/ce100-app#1495]
Tags are styled differently across the app
![screen shot 2018-09-21 at 12 10 02](https://user-images.githubusercontent.com/20577339/45878017-8bebd500-bd97-11e8-8b46-f025428e3151.png)

**Comment by fannyBkane** [comment]
@asimonw @Kimberley-Heath good to keep on the radar if this can be prioritised

**Comment by asimonw** [comment]
@fannyBkane I agree. We could simply turn this HMW into an issue into the sprint milestone?

**Comment by fannyBkane** [comment]
@asimonw sounds good 👍
